### ISSUE TITLE
Generating PBN-output with PAR-result

### ISSUE THREAD

**Issue by ThorvaldAagaard** [dealerv2/Dealer-Version-2-#17]
I would like to include the PAR-result for the deal, when I generate it.

It is not clear in the documentation and the examples if this is possible.

**Comment by dealerv2** [comment]
yes it is. run dealerv2 with the following input (interactively if u like)

dealerv2
produce 3
action printrpt( deal, par(NS,ns), parcontract(ns) )
CTL-D
will produce the following output:
In ParScore:: Using Parscore requires TableMode. Setting Mode and continuing..
 n QJT54..KJT.AQ962 e AK732.AJ653.6.87 s 8.K9842.Q743.JT5 w 96.QT7.A9852.K43 ,100,' 3H* by EW ns'
 n Q5.Q52.AQJ86.QT5 e J9763.T986.73.A3 s AK842.AJ3.KT4.84 w T.K74.952.KJ9762 ,630,' 3N  by NS ns'
 n QJ8654.Q95.T64.9 e 972.KJ4.QJ752.K7 s K.A87632.3.AJT32 w AT3.T.AK98.Q8654 ,-200,' 4H* by NS ns'
Generated 3 hands
Produced  3 hands
Initial random seed 64227441979653
Time needed   46.435 sec

I am not sure if the version you have supports the parcontract (probably not) or the new version of the par keyword that lets you enter the vulnerability. But the new one will as soon as I upload it.
the current one should work tho if you just use the printrpt(deal, par(ns) ) action.
JGM

________________________________
From: Thorvald Aagaard ***@***.***>
Sent: 06 April 2024 02:12
To: dealerv2/Dealer-Version-2- ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [dealerv2/Dealer-Version-2-] Generating PBN-output with PAR-result (Issue #17)


I would like to include the PAR-result for the deal, when I generate it.

It is not clear in the documentation and the examples if this is possible.
